pkg/ingester/ingester.go

```diff
@@ -32,13 +32,16 @@ const (
 	outOfOrderTimestamp     = "timestamp_out_of_order"
 	duplicateSample         = "multiple_values_for_timestamp"
 	greaterThanMaxSampleAge = "greater_than_max_sample_age"
+	maxLabelNamesPerSeries  = "max_label_names_per_series"
 
 	// DefaultConcurrentFlush is the number of series to flush concurrently
 	DefaultConcurrentFlush = 50
 	// DefaultMaxSeriesPerUser is the maximum number of series allowed per user.
 	DefaultMaxSeriesPerUser = 5000000
 	// DefaultMaxSeriesPerMetric is the maximum number of series in one metric (of a single user).
 	DefaultMaxSeriesPerMetric = 50000
+	// DefaultMaxLabelNamesPerSeries is the maximum number of label names in one series.
```

Comment:
I think this should be "per metric".

Reply:
Series for the same metric 'can' have different labels, although it is rare.  I think series is correct here.